config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java

```diff
@@ -295,6 +295,19 @@ public void testImportingFromStoredExpressionsWithSmallConstants() throws IOExce
         }
```

Comment:
Could you verify that the stored macro is created and have the expected expression here?

Reply:
You mean like we do in the next test, `testMacroGeneration`?

Reply:
Yes, but in the re-loaded model.
In a real system the model is built from file, then later rebuilt from ZooKeeper at which point the original TensorFlow model is not available. The "importFromStored" tests (only) recreates this situation, so I think one of those need to verify that the macros are present and correct after re-loading from ZooKeeper data.

Reply:
Ahh I see, thanks.